pkg/kubelet/dockertools/manager.go

```diff
@@ -1225,14 +1225,19 @@ func (dm *DockerManager) createPodInfraContainer(pod *api.Pod) (kubeletTypes.Doc
 			}
 			return "", err
 		}
+		if ref != nil {
+			dm.recorder.Eventf(ref, "pulled", "Successfully pulled Pod container image %q", container.Image)
+		}
 	}
-	if ref != nil {
-		dm.recorder.Eventf(ref, "pulled", "Successfully pulled image %q", container.Image)
+	if ok && ref != nil {
+		dm.recorder.Eventf(ref, "pulled", "Pod container image %q already present on machine", container.Image)
 	}
 
 	id, err := dm.runContainerInPod(pod, container, netNamespace, "")
 	if err != nil {
 		return "", err
+	} else {
+		dm.recorder.Eventf(ref, "started", "Started Pod Infra Container")
```

Comment:
Yeah, but I found it useful to log explicitly start of an infra container. Otherwise there's quite a lot o noise.